devtools/pgmocktime/mocker.go

```diff
@@ -156,6 +156,12 @@ func (m *Mocker) AdvanceTime(ctx context.Context, d time.Duration) error {
 	return m.readErr("advance time")
 }
 
+// AdvanceTime advances the time by the given duration in seconds (workaround for pgmocktime milliseconds interval field value out of range error (~2147520000 milliseconds)).
```

Comment:
I'm thinking we just fix the bug in `AdvanceTime` rather than creating a new method that isn't broken -- we can do so without changing behavior.

Maybe break it down into hours + milliseconds

`base_time = %s.now() + '%d hours'::interval + '%d milliseconds'::interval`